docs/pages/preview/upcoming-releases.mdx

```diff
@@ -4,76 +4,60 @@ description: A timeline of upcoming Teleport releases.
 h1: Upcoming Teleport Releases
 ---
 
-The teleport team delivers a new major release roughly every 3 months.
+The Teleport team delivers a new major release roughly every 3 months.
 
 ## Teleport
 
-### Teleport 8.3
+### Teleport 11.1
 
-Teleport 8.3 will add support for IAM based joining on AWS and Session Moderation.
+Teleport 11 will bring support for Windows Hello for per-session MFA, passwordless
+and Login Rules, script-based automatic upgrades.
 
-https://github.com/gravitational/teleport/milestone/69
+- FIPS support to Desktop Access
+- Per-session MFA for Teleport Connect
+- Resource-based Access Requests for Teleport Plugins
+- IP-based validation for Kubernetes Access
+- Improved Microsoft SQL Server integration for Database Access
 
-| Version     | Date               | Description                      |
-| ----------- | ------------------ | -------------------------------- |
-| Release     | February 14th, 2022 | Good to go for production.      |
+| Version          | Date          | Description                                                 |
+|------------------|---------------|-------------------------------------------------------------|
+| Release          | November 2022 | Good to go for production.                                  |
 
-You can find the full list of fixes and features in the [GitHub milestone](https://github.com/gravitational/teleport/milestone/69).
+### Teleport 12
 
-### Teleport 9
-
-Teleport 9.0 will focus on bringing Desktop Access to GA with support for
-session recording, per-session MFA, and clipboard support. Database Access
-will add self-hosted Redis, auto-discovery for Redshift clusers, and
-auto-IAM configuration improvements.
-
-In addition, Teleport 9 will include preview releases of Microsoft SQL
-Server with AD authentication for Database Access and Teleport Certificate Bot.
-
-| Version     | Date               | Description                      |
-| ----------- | ------------------ | -------------------------------- |
-| Alpha       | Febuary 14th, 2022 | Good for some testing and demos. |
-| Beta        | Febuary 21st, 2022 | Deploy on staging.               |
-| Release     | March 11th, 2022   | Good to go for production.       |
-
-You can find the full list of fixes and features in the [GitHub milestone](https://github.com/gravitational/teleport/milestone/64).
+Teleport 12 will focus on support for additional Clouds (GCP and Azure),
+Device Trust, improved resource onboarding, TUF-based Automatic Upgrades, and
```

Comment:
Is Automatic Upgrades a Teleport product, or should we use "automatic upgrades" instead?